src/app/zap-templates/zcl/data-model/chip/sample-mei-cluster.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <domain>General</domain>
+    <name>Sample MEI</name>
+    <code>0xFFF1FC20</code>
```

Comment:
Since this is a sample, it would probably be helpful to indicate where the prefix of this command is coming from. 

If the vendor ID is changed, does this also need to change?

Reply:
Comment added. I'll push it when the current CI finishes